parser.go

```diff
@@ -19042,6 +19042,18 @@ yynewstate:
 				if lexer, ok := yylex.(stmtTexter); ok {
 					s.SetText(parser.lexer.client, lexer.stmtText())
 				}
+
+				// fill arana hints
+				if hints := parser.lexer.aranaHints; len(hints) > 0 {
+					type hintSetter interface {
```

Comment:
Can the declaration of this hintSetter interface be placed in a public place (suggestions only)?

Reply:
👍